pyuvdata/parameter.py

```diff
@@ -323,7 +323,7 @@ def lat_lon_alt(self):
         if self.value is None:
             return None
         else:
-            return utils.LatLonAlt_from_XYZ(self.value)
+            return utils.LatLonAlt_from_XYZ(self.value, check_acceptability=False)
```

Comment:
hard coding this here needs to happen b/c of the way in which `LocationParameter` decorates `UVBase` attributes. this is fine, because `UVBase` has its own set of acceptability checks on its relevant attributes that are more easily exposed to the end-user.

Reply:
I think this is fine, but can you add a comment to this effect so people reading the code in the future understand why it's defaulted this way? 

I think you need to make this same change to the `lat_lon_alt_degrees` method as well.